docs/content/configuration/reference.md

```diff
@@ -29,22 +29,135 @@ relative to the configuration file directory the path is defined
 within.
 :::
 
-## Index
-
-[`anonymizer_paths`](#anonymizer-paths) |
-[`backup_binary`](#backup-binary) |
-[`backup_excluded_tables`](#backup-excluded-tables) |
-[`backup_expiration_age`](#backup-expiration-age) |
-[`backup_options`](#backup-options) |
-[`backup_timeout`](#backup-timeout) |
-[`connections` (standalone)](#connections) |
-[`default_connection`](#default-connection) |
-[`restore_binary`](#restore-binary) |
-[`restore_options`](#restore-options) |
-[`restore_timeout`](#restore-timeout) |
-[`storage_directory`](#storage-directory) |
-[`storage_filename_strategy`](#storage-filename-strategy) |
-[`workdir`](#workdir)
+<div class="toc-inline">
+
+  [[toc]]
+
+</div>
+
+<style module>
+.toc-inline .table-of-contents ul {
+  list-style: none;
+  display: flex;
+}
+.toc-inline .table-of-contents li {
+  flex: 1;
+  display: inline-block;
+}
+</style>
+
+## `anonymization`
+
+Write the anonymization configuration directly in the main configuration file.
+
+Keys under the first dimension are connections names, then follows the structure
+expected in anonymization configuration files.
+
```

Comment:
Maybe add something about that if there's only one connection and no name is given, the connection is called `default`

Reply:
I added it in the `connections` configuration, but it may be repeated here.

Or maybe we should add an *Essentials* section on top, which explains how are handled all file paths, connections names, etc... ?

Reply:
I don't know, I think that what make me sad here is that we have to precise `default` even if we only had one connection ^^

But there is no easy solution to this :grimacing:

Reply:
Hum, I just want to merge this. We are going to continue improving the documentation in the next few days or weeks.

Reply:
I was adding a tip specifying this, but I realized I would have to add it everywhere we use connection names in configuration.

That's not that much sections, but it's a lot of copy/pasted pieces...

Reply:
@SimonMellerin That's only 2, so I did it, is that OK for you ?

Reply:
thanks !